src/features/mcda/components/MCDALayerEditor/MCDALayerParameters/MCDALayerParameters.tsx

```diff
@@ -0,0 +1,385 @@
+import { Edit16 } from '@konturio/default-icons';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { Button, Input, Select, Text } from '@konturio/ui-kit';
+import { useAtom } from '@reatom/npm-react';
+import clsx from 'clsx';
+import { i18n } from '~core/localization';
+import { TooltipTrigger } from '~components/TooltipTrigger';
+import { LAYERS_PANEL_FEATURE_ID } from '~features/layers_panel/constants';
+import { isNumber } from '~utils/common';
+import { bivariateStatisticsResourceAtom } from '~core/resources/bivariateStatisticsResource';
+import { Sentiments } from '../Sentiments';
+import { MCDALayerParameterRow } from './MCDALayerParameterRow/MCDALayerParameterRow';
+import s from './MCDALayerParameters.module.css';
+import {
+  MCDA_LAYER_DEFAULTS as DEFAULTS,
+  NUMBER_FILTER,
+  POSITIVE_NUMBER_FILTER,
+  SENTIMENT_VALUES,
+  normalizationOptions,
+  outliersOptions,
+  sentimentColors,
+  sentimentsOptions,
+  transformOptions,
+} from './constants';
+import type {
+  MCDALayer,
+  TransformationFunction,
+} from '~core/logical_layers/renderers/stylesConfigs/mcda/types';
+import type { Normalization } from '~core/logical_layers/renderers/stylesConfigs/mcda/types';
+
+export type MCDALayerLegendProps = {
+  layer: MCDALayer;
+  onLayerEdited: (editedMCDALayer: MCDALayer) => void;
+};
+
+export function MCDALayerParameters({ layer, onLayerEdited }: MCDALayerLegendProps) {
+  const [editMode, setEditMode] = useState(false);
+  const [sentiment, setSentiment] = useState(DEFAULTS.sentiment as string);
+  const [rangeFrom, setRangeFrom] = useState(DEFAULTS.range[0]);
```

Comment:
Is it better to store ranges with single useState?

`const [[rangeFrom, rangeTo], setRanges] = useState(DEFAULTS.range)`

Reply:
> Is it better to store ranges with single useState?
> 
> `const [[rangeFrom, rangeTo], setRanges] = useState(DEFAULTS.range)`

I made it an array at first.
Array makes it a bit easier to combine data into single object, but harder to do error checks and display separate errors. We still need to display and highlight empty/invalid input fields separately here.

I think think with two fields it doesn't matter. If there was a lot of them, array could be better